draft-packages/tag/KaizenDraft/Tag/Tag.scss

```diff
@@ -43,20 +42,20 @@ $small: $spacing-md;
 }
 
 .validationIcon {
-  @include ca-margin($start: calc(#{$spacing-md} * -0.15), $end: $spacing-xs);
-  display: flex;
   align-items: center;
+  display: flex;
+  margin-inline-start: calc(#{$spacing-md} * -0.15);
+  margin-inline-end: $spacing-xs;
 }
 
 .profile {
-  @include ca-margin($start: calc(#{$spacing-md} * -0.15), $end: $spacing-xs);
+  margin-inline-start: calc(#{$spacing-md} * -0.15);
+  margin-inline-end: $spacing-xs;
   display: flex;
   align-items: center;
 }
 
 .textContent {
-  @include kz-typography-paragraph-small;
-  @include ca-inherit-baseline;
```

Comment:
This definitely isn't needed, and nothing needs to be put in as a replacement?

Reply:
Not if we're using <Paraghraph> - especially not ca-inherit-baseline - that's a super old pattern that should have long been deprecated